pbjson-test/src/lib.rs

```diff
@@ -32,6 +32,291 @@ pub mod test {
         include!(concat!(env!("OUT_DIR"), "/test.common.rs"));
         include!(concat!(env!("OUT_DIR"), "/test.common.serde.rs"));
     }
+    pub mod skipped_tests {
+        pub mod syntax3 {
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.syntax3.rs"));
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.syntax3.serde.rs"));
+        }
+        pub mod common {
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.common.rs"));
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.common.serde.rs"));
+        }
+    }
+}
+
+#[cfg(test)]
+mod skipped_tests {
+    use super::*;
+    use crate::test::syntax3::kitchen_sink::MixedCase;
+    use chrono::TimeZone;
+    use pbjson_types::{Duration, Timestamp};
+    use test::skipped_tests::syntax3::*;
+
+    #[test]
+    fn test_empty() {
+        let message = Empty {};
+
+        let encoded = serde_json::to_string(&message).unwrap();
+        let _decoded: Empty = serde_json::from_str(&encoded).unwrap();
+
+        let empty = serde_json::from_str::<Empty>("{\n \"foo\": \"bar\"\n}").unwrap();
+        assert_eq!(empty, Empty {});
+    }
+
```

Comment:
I think a test of the unknown fields behaviour both with the feature enabled and disabled would be good. _Ideally we would already have a test of the latter case but sadly we do not_

Reply:
Yeah, I think testing the empty behaviour is important, as it is a branch in the code generation, I was suggesting another thing we might test